src/config.rs

```diff
@@ -1,11 +1,12 @@
 use color_eyre::Result;
 use dotenvy::dotenv;
 use serde::Deserialize;
+use argon2::{self, Config, ThreadMode, Variant, Version};
```

Comment:
Порядок импортов соблюсти
```suggestion
use argon2::{self, Config, ThreadMode, Variant, Version};
use color_eyre::Result;
use dotenvy::dotenv;
use serde::Deserialize;
```